compiler/optimizer/RegisterCandidate.cpp

```diff
@@ -955,6 +955,14 @@ TR_RegisterCandidate::processLiveOnEntryBlocks(TR::Block * * blocks, int32_t *bl
          if (trace)
             traceMsg(comp, "\tExamining live-on-entry block_%d with loads and stores = %d\n", blockNumber, (*_loadsAndStores)[blockNumber]);
 
+         // Ignore uses in OSR induce blocks
+         if (blocks[blockNumber]->isOSRInduceBlock(comp))
+            {
+            _liveOnEntry.reset(blockNumber);
```

Comment:
Do we need to reset the live on entry for the induce OSR block ? 

I understand the reasoning behind the "continue" statement skipping the weighing process in this change, but resetting live on entry for the induce OSR block could lead to stores being emitted on the main line path before the OSR guard which is undesirable.